tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -595,6 +599,14 @@ public static MsalTokenResponse CreateAadTestTokenResponseWithFoci()
             return msalTokenResponse;
         }
 
+        public static MsalTokenResponse CreateAadTestTokenResponseWithMsalUserDefault()

```

Comment:
Shouldn't this be in [MosckHelpers.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/09155f3499b6d77a56ae915dd150f4db8dcb8002/tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs#L27) ?